pkgs/top-level/go-packages.nix

```diff
@@ -3765,7 +3765,12 @@ let
     sha256 = "1mf98hagb0yp40g2mbar7aw7hmpq01clnil6y9khvykrb33vy0nb";
 
     postInstall = ''
-      for i in $bin/bin/{provider,provisioner}-*; do mv $i $bin/bin/terraform-$(basename $i); done
+      # prefix all the plugins with "terraform-"
+      for i in $bin/bin/*; do
```

Comment:
Or keep it simple :)

On Sun, 15 May 2016 11:00 Joachim Fasting, <notifications@github.com> wrote:

> In pkgs/top-level/go-packages.nix
> <https://github.com/NixOS/nixpkgs/pull/15455#discussion_r63289263>:
>
> > @@ -3765,7 +3765,12 @@ let
> >      sha256 = "1mf98hagb0yp40g2mbar7aw7hmpq01clnil6y9khvykrb33vy0nb";
> >
> >      postInstall = ''
> > -      for i in $bin/bin/{provider,provisioner}-*; do mv $i $bin/bin/terraform-$(basename $i); done
> > +      # prefix all the plugins with "terraform-"
> > +      for i in $bin/bin/*; do
>
> You could always use a sub-shell [image: :grinning:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/15455/files/4188f2c91d8e6f50a7960e974932f459918578c0#r63289263>
>